src/events/config.go

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright © 2020, Oracle and/or its affiliates. All rights reserved.
+ * The Universal Permissive License (UPL), Version 1.0
+ */
+package events
+
+import (
+	"os"
+
+	"github.com/oracle/oci-go-sdk/common"
+	"github.com/oracle/oci-go-sdk/streaming"
+)
+
+// EnvironmentConfigurationProvider uses environment variables to get OCI config
+func EnvironmentConfigurationProvider() (common.ConfigurationProvider, error) {
+	pass := os.Getenv("PASSPHRASE")
```

Comment:
any reason for `pass` being outside? (is it because it can be empty?)

Reply:
because the function signature requires a pointer: https://github.com/oracle/oci-go-sdk/blob/master/common/configuration.go#L55